src/java.desktop/share/classes/java/awt/Robot.java

```diff
@@ -126,6 +126,16 @@ public class Robot {
 
     private DirectColorModel screenCapCM = null;
 
+    /**
+     * Default step-delay in milliseconds for mouse {@link #glide(int, int, int, int) glide}.
+     */
+    public final int DEFAULT_STEP_DELAY = 20;
+
+    /**
+     * Default pixel step-length for mouse {@link #glide(int, int, int, int) glide}.
+     */
+    public final int DEFAULT_STEP_LENGTH = 2;
```

Comment:
Is there a reason these fields are not static? Non-static final fields like these are constant only in the class and not constant when used by other classes, and require an instance to access, but I don't find these differences convincing as the reason.

Reply:
Updated. Will update in the CSR as well. Thanks!